about.html

```diff
@@ -37,7 +37,7 @@
           <a href="">&times;</a>
           <a href="index.html" class="close-menu">Home</a>
           <a href="about.html" class="close-menu">About Us</a>
-          <a href="#" class="close-menu">Inquirires</a>
+          <a href="" class="close-menu">Inquirires</a>
           <a href="#" class="close-menu">Partner</a>
        </div>
```

Comment:
- [ ] This section 👇🏼 does not exist, kindly considder removing it from the About page

![Screenshot_20230706_202756](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/7c7bfe9c-4923-4ff2-9c56-5c0ae210a10e)